jwst/resample/resample.py

```diff
@@ -285,7 +285,7 @@ def resample_variance_array(self, name, output_model):
             inwht = resample_utils.build_driz_weight(
                 model,
                 weight_type=None,
-                good_bits="~NON_SCIENCE+REFERENCE_PIXEL"
+                good_bits=self.good_bits
```

Comment:
So for the specific case of something like MIRI imager, which also contains data from the coronagraphic mask areas, the coronagraphic areas are not included in the drizzled SCI image because they have something set in `good_bits` that excludes them and hence by also applying `good_bits` to the variance/weight array they'll get excluded from that too?

Reply:
Exactly. In any case, the science image only considers these good_bits anyway, so the error array should do the same to be consistent